### PR TITLE
MBS-13370: Rewrite more artist pages for Genie

### PR DESCRIPTION
**Pull request by Maxr1998**
# Problem

Genie has multiple tabs on their artist pages, the main artist page uses the format https://genie.co.kr/detail/artistInfo?xxnm=81599561, while other pages showing albums, songs and MVs differ slightly (e.g., https://genie.co.kr/detail/artistAlbum?xxnm=81599561).

# Solution

They should be rewritten to the (canonical) main artist page so that relationship types are auto-selected.

Closes MBS-13370.

# Testing

Added an extra test case that rewrites the albums page for an artist to their main page.

### CODE REVIEW

File: root/static/scripts/edit/URLCleanup.js [hunk 1]
```diff
@@ -2674,6 +2674,7 @@ const CLEANUPS: CleanupEntries = {
     },
     clean: function (url) {
       url = url.replace(/^(?:https?:\/\/)?(?:(?:www|mw)\.)?genie\.co\.kr\//, 'https://www.genie.co.kr/');
+      url = url.replace(/^https:\/\/www\.genie\.co\.kr\/detail\/artist(?:Info|Album|Song|Mv)\?xxnm=(\d+)(?:[&#\/].*)?$/, 'https://www.genie.co.kr/detail/artistInfo?xxnm=$1');
       url = url.replace(/^(https:\/\/www\.genie\.co\.kr\/detail\/(?:albumInfo\?ax|artistInfo\?xx|songInfo\?xg|mediaInfo\?xv)nm=\d+)(?:[&#\/].*)?$/, '$1');
```
Comment (reosarevok) [inline root/static/scripts/edit/URLCleanup.js:2678]: Both of these look at `artistInfo` - should we drop it from the second line? Is it needed twice?

Are there similar multiple pages for album, song, media?

Comment (Maxr1998) [reply root/static/scripts/edit/URLCleanup.js:2678]: I think we can remove it from the second line, yes.
There are tabs on album and song details, but all those other tabs redirect to the artist page, so they're already handled by this.

Comment (Maxr1998) [reply root/static/scripts/edit/URLCleanup.js]: Cleaned up the RegEx, waiting for the tests to pass now.

**Review by reosarevok — APPROVED** [review]
Seems good now.

**Review by yvanzo — APPROVED** [review]
LGTMBDNT. Thank you again!

**Pull request merged**